proxies/live/apiproxy/targets/apim.xml

```diff
@@ -3,5 +3,8 @@
   <PreFlow/>
   <HTTPTargetConnection>
     {{ HOSTED_TARGET_CONNECTION }}
+    <Properties>
+      <Property name="io.timeout.millis">65000</Property>
+  </Properties>
```

Comment:
Very pedantic request, sorry. But can you make the tabs match for Properties? Other than that all good